pom-redhat.xml

```diff
@@ -48,34 +48,300 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+
       <dependency>
         <groupId>org.wildfly.swarm</groupId>
         <artifactId>bom-certified</artifactId>
         <version>${version.wildfly.swarm}</version>
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+
       <dependency>
         <groupId>org.keycloak</groupId>
         <artifactId>keycloak-authz-client</artifactId>
-        <version>2.5.7.Final-redhat-2</version>
+        <version>2.5.5.Final</version>
```

Comment:
This is `pom-redhat.xml`, we probably want to use the productized Keycloak stuff?

Reply:
No, this has been discussed with Ken and we step back from the productised bits here to make it more "certified"

Reply:
Oh, this is the "tech preview" stuff, right?